### ISSUE TITLE
an engine's default configuration is flagged as an error

### ISSUE THREAD

**Issue by efx** [ember-cli/eslint-plugin-ember#251]
Out of the box, the avoid leaking state rules show an error for an engine's configuration:
```js
const Eng = Engine.extend({
  modulePrefix,
  Resolver,
  dependencies: {
    services: [
      'foo'
    ]
  }
});
```
This is similar to https://github.com/ember-cli/eslint-plugin-ember/issues/237.
I think this is a problem for onboarding developers with engines.

**Comment by bertdeblock** [comment]
Any suggestions on how to fix this? I was thinking we could ignore variables that have been `require`d, but I'm very new to writing lint rules so I don't know if that's a good/solid idea and how future-proof it would be.

**Comment by bertdeblock** [comment]
You can ignore my comment I think (though it's related). This issue is about the `addon/engine.js` file. I was also running into this issue for the top level `index.js` file of an engine.
